server/runtime/src/main/server/bin/cli.sh

```diff
@@ -122,16 +127,16 @@ if $linux; then
     for var in $CONSOLIDATED_OPTS
     do
        # Remove quotes
-       p=$(echo $var | tr -d "'")
+       p=$(echo "$var" | tr -d "'")
        case $p in
-         -Dinfinispan.server.base.dir=*)
```

Comment:
If not used I would remove ispn.server.*.path use

Reply:
There are some commands that reference those paths: `user`, `credential`, so I'd rather leave those in